app/routes/funding/what-funding.js

```diff
@@ -24,35 +27,37 @@ module.exports = [{
   options: {
     validate: {
       payload: Joi.object({
-        standard: Joi.array().items(Joi.string()).single()
+        standard: Joi.required()
       }),
       failAction: async (request, h, error) => {
         const { agreement } = await cache.get(request)
         const { funding } = agreement
-        const eligibleFunding = await getFunding(request)
+        const eligibleFunding = await getFunding(request) ?? undefined
         if (!eligibleFunding) {
           return h.view('no-response').takeover()
         }
         return h.view('funding/what-funding', new ViewModel(eligibleFunding, funding, error)).code(400).takeover()
       }
     },
     handler: async (request, h) => {
-      const { standard } = request.payload
+      const standards = typeof (request.payload.standard) === 'string' ? [request.payload.standard] : request.payload.standard
```

Comment:
Knock-on effect from Joi not validating the standard payload is an array it has to explicitly check and cast if needs be